components/PlantForm.js

```diff
@@ -0,0 +1,206 @@
+import styled, { css } from "styled-components";
+import { StyledButton } from "./StyledButton";
+
+const StyledFormWrapper = styled.form`
+  display: flex;
+  flex-direction: column;
+
+  ${css`
+    input[type="text"] {
+      width: 100%;
+      padding: 15px;
+      border-style: none;
+      border-radius: 10px;
+    }
+    textarea {
+      width: 100%;
+      padding: 15px;
+      border-style: none;
+      border-radius: 10px;
+      font-family: Arial, Helvetica, sans-serif;
+    }
+    h3 {
+      margin-bottom: 10px;
+    }
+    section {
+      display: flex;
+      flex-direction: row;
+      justify-content: start;
+    }
+    label {
+      /* genaue Anpassung in einer späteren User-Story (nach Wahl der Schriftart, Größe, etc.) */
+      margin-right: 15px;
+    }
+
+    input {
+      margin: 0 5px 0 0;
+    }
+
+    div.button {
+      margin-top: 15px;
+      display: flex;
+      justify-content: center;
+    }
+  `}
+`;
+
+export default function PlantForm({ onSubmitCreatePlant }) {
+  function handleSubmit(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+    data.fertiliserSeason = formData.getAll("fertiliserSeason");
+    onSubmitCreatePlant(data);
+    event.target.reset();
+  }
+
+  return (
+    <>
+      <StyledFormWrapper onSubmit={handleSubmit}>
+        <label htmlFor="name">
+          <h3>Plant Name: *</h3>
+        </label>
+        <input
+          type="text"
+          id="name"
+          name="name"
+          placeholder="Plant Name"
+          required
+        />
+
+        <label htmlFor="botanicalName">
+          <h3>Botanical Name: *</h3>
+        </label>
+        <input
+          type="text"
+          id="botanicalName"
+          name="botanicalName"
+          placeholder="Botanical Name"
+          required
+        />
+
+        <label htmlFor="description">
+          <h3>Description:</h3>
+        </label>
+        <textarea
+          id="description"
+          name="description"
+          rows="5"
+          placeholder="Plant Description"
+        ></textarea>
+
+        <label htmlFor="lightNeed">
+          <h3>Light Need: *</h3>
+        </label>
+
+        <section>
```

Comment:
_Refactor suggestion (for now or a future improvement):_ 

Consider mapping over an array of light options to dynamically generate the radio inputs. This reduces repetition and makes it easier to update the options in the future.

Example

```js
const lightOptions = [
  { id: "lightNeed1", value: "full sun", label: "Full Sun" },
  { id: "lightNeed2", value: "partial shade", label: "Partial Shade" },
  { id: "lightNeed3", value: "full shade", label: "Full Shade" },
];

<section>
  {lightOptions.map((option) => (
    <div key={option.id}>
      <input
        type="radio"
        id={option.id}
        name="lightNeed"
        value={option.value}
        required={option.id === "lightNeed1"} 
      />
      <label htmlFor={option.id}>{option.label}</label>
    </div>
  ))}
</section>

```

Reply:
thank you, is implented